types/material-ui/index.d.ts

```diff
@@ -1846,6 +1846,7 @@ declare namespace __MaterialUI {
         autoFocus?: boolean;
         min?: number;
         max?: number;
+        maxLength?: string;
```

Comment:
If you're referring to HTML `input`'s [maxlength](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input), Please note that it's not `maxLength` it's `maxlength`. Also, please add `minlength` too.